src/database/index.ts

```diff
@@ -0,0 +1,36 @@
+import { Client, Pool } from "pg";
+export const DB_NAME = "mydb";
```

Comment:
please use meaningful name for DB_NAME and put it in somewhere safe (like in .env file)

Reply:
this can lead to security issues